src/Command/RunCommand.php

```diff
@@ -7,39 +7,55 @@
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Yiisoft\Yii\Console\ExitCode;
-use Yiisoft\Yii\Queue\QueueFactory;
 use Yiisoft\Yii\Queue\QueueFactoryInterface;
 
 final class RunCommand extends Command
 {
     protected static $defaultName = 'queue/run';
-    protected static $defaultDescription = 'Runs all the existing messages in the queue. Exits once messages are over.';
+    protected static $defaultDescription = 'Runs all the existing messages in the given queues. ' .
+        'Exits once messages are over.';
 
     private QueueFactoryInterface $queueFactory;
+    private array $channels;
 
-    public function __construct(QueueFactoryInterface $queueFactory)
+    public function __construct(QueueFactoryInterface $queueFactory, array $channels)
     {
         parent::__construct();
+
         $this->queueFactory = $queueFactory;
+        $this->channels = $channels;
     }
 
     public function configure(): void
     {
         $this->addArgument(
             'channel',
-            InputArgument::OPTIONAL,
-            'Queue channel name to connect to',
-            QueueFactory::DEFAULT_CHANNEL_NAME
-        );
+            InputArgument::OPTIONAL | InputArgument::IS_ARRAY,
```

Comment:
Channel is required options.

Reply:
Not anymore, see another changes in this PR.